src/ripple/app/misc/impl/AMMUtils.cpp

```diff
@@ -138,6 +138,10 @@ std::uint16_t
 getTradingFee(ReadView const& view, SLE const& ammSle, AccountID const& account)
 {
     using namespace std::chrono;
+    // should not happen
```

Comment:
Asserts by definition "should not happen". The comment doesn't add value. I'd remove it.